src/renderers/webvr/WebVRManager.js

```diff
@@ -137,6 +154,11 @@ function WebVRManager( renderer ) {
 		cameraL.matrixWorldInverse.fromArray( frameData.leftViewMatrix );
 		cameraR.matrixWorldInverse.fromArray( frameData.rightViewMatrix );
 
+		if ( stageParameters ) {
```

Comment:
As we already will have an `standingMatrix` no matter if from `stageParameters` or created using `userHeight` this should not be an if()